yarn.lock

```diff
@@ -1173,16 +1173,18 @@
     proxy-from-env "^1.1.0"
     which "^2.0.2"
 
-"@sentry/cli@^1.52.4":
-  version "1.57.0"
-  resolved "https://registry.yarnpkg.com/@sentry/cli/-/cli-1.57.0.tgz#87237ec6b77cb7582f018b580e9bb9481a809baa"
-  integrity sha512-lgfSQMRVcf00IkHkmqY6L9pv1uOEKDKbEjFQu2qstjg3kycswo1prz60xORJVpfr4qXDzqbSsAKEBp4VKG+ilw==
+"@sentry/cli@^1.72.0":
```

Comment:
Are you sticking with `1.72.0` because of the wizard dependency?

Reply:
Yeah, that's the min. version of the CLI for the newer wizard, but the `^` means anything above it, it's not 1.72 only.